### DISCUSSION TITLE
Replay Gain - Add/Subtract from TAG target value

### DISCUSSION THREAD

**Discussion by StillNotWorking** [LMS-Community/slimserver#586] [Ideas]
Discussions are often to scan library for -18 LUFS or -23 LUFS. While some like to aim for -14 LUFS.

If one could scan library for the usual EBU R128/ITU BS.1770 standard (-18 LUFS). But then be able to adjust to other targets from LMS Replay Gain settings would be a nice feature.

Or if someone already have scanned for -23 LUFS but then figure out thay want to add 5 LUFS for -18 LUFS target.

If someone know if this already can be done with som file editing please let us know :)

thans!

**Comment by mherger** [comment]
Hi @StillNotWorking - I believe you'd get more/better feedback by posting to [forums.slimdevices.com](forums.slimdevices.com).
